add_namespaces.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+cd "Assets/Classes/"
+
+# Append text to beginning of each cs file
```

Comment:
Why do you have this code commented out?

Reply:
> For instance, if you find something new in an XML that there is no class for, what happens now?

The `XmlSerializer` constructor has some options for how it handles unknown XML elements/attributes. I haven't touched it yet since I know all of the tags I'll be working with. We should definitely figure out some of the schema from John Caley when we get the chance. I think translating those missing elements to a different tag would be difficult but there's some pretty automated ways to discard the unknown tags.

I also think we could handle the validation in the `CaveWriting-Projects` script. Working with XML in Python has been surprisingly easy and we'll be combing through the XML anyways

Reply:
I commented out the code just so I don't accidentally run something that I don't mean to. It's not smart enough to tell if the namespaces have already been added and would be rather a headache to undo. I could add it to the gitignore if you think it's warrented?